src/checkers/inference/solver/backend/AbstractFormatTranslator.java

```diff
@@ -24,15 +24,18 @@
 import checkers.inference.util.ConstraintVerifier;
 
 /**
- * Created by mier on 08/11/17.
+ * Abstract implementation of FormatTranslator that all concrete FormatTranslator should extend from.
+ * It has the necessary lattice that will be used and already setup the abstract structure and provided
+ * default implementations so that you only need to override minimum methods your FormatTranslator is
+ * interested.
  */
 public abstract class AbstractFormatTranslator<SlotEncodingT, ConstraintEncodingT, SlotSolutionT>
         implements FormatTranslator<SlotEncodingT, ConstraintEncodingT, SlotSolutionT>{
 
-    // lattice is protected because every subclass of AbstractFormatTranslator uses it.
+    /* lattice is protected because every subclass of AbstractFormatTranslator uses it.*/
```

Comment:
Same, Instead of explaining the visibility modifiers, document what this field is. For lattice, we can simply document it as something like:
```
/**
* Target lattice for the type system.
*/